wrappers.lisp

```diff
@@ -184,24 +89,9 @@
               :name nil :type nil :defaults pathname))))
     (file-error () nil)))
 
-(defun portable-file-write-date (pathname)
-  ;; clisp errors out if you run file-write-date on a directory.
-  #+clisp (posix:file-stat-mtime (posix:file-stat pathname))
-  #-clisp (file-write-date pathname))
-
 (defun print-backtrace (&optional (frames 100))
   "print a backtrace of FRAMES number of frames to standard-output"
-  #+sbcl (sb-debug:print-backtrace :count frames :stream *standard-output*)
-  #+clisp (ext:show-stack 1 frames (sys::the-frame))
-  #+ccl (ccl:print-call-history :count frames :stream *standard-output* :detailed-p nil)
-  ;; borrowed from 'trivial-backtrace'
-  #+lispworks (let ((dbg::*debugger-stack*
-                      (dbg::grab-stack nil :how-many frames))
-                    (*debug-io* *standard-output*)
-                    (dbg:*debug-print-level* nil)
-                    (dbg:*debug-print-length* nil))
-                (dbg:bug-backtrace nil))
-  #-(or sbcl clisp ccl lispworks) (write-line "Sorry, no backtrace for you."))
+  (sb-debug:print-backtrace :count frames :stream *standard-output*))
 
 (defun bytes-to-string (data)
```

Comment:
Yes. Because it is not as straight forward how to clean them up. It may be possible to only leave to functions utf8-to-octets and octets-to-utf8, but I haven't gone through their uses yet. See issue #320